docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,448 @@
+# Initialize Vendor Accounts
```

Comment:
@waxlamp @yarikoptic @satra @kabilar would be great if you could review

Reply:
Needs Dockerhub (or some other container registry)